external/dynamo/start_dynamo_unified_thompson_hints.sh

```
@@ -272,41 +273,64 @@ docker run -d \
             if ! kill -0 \$pid 2>/dev/null; then
                 echo \"ERROR: \$worker_type worker process died!\"
                 return 1
             fi
-            
-            sleep 5
-            elapsed=\$((elapsed + 5))
-            if [ \$((elapsed % 15)) -eq 0 ]; then
-                echo \"  ... \${elapsed}s / \${max_wait}s\"
+
+            # Check ETCD for registered workers using v3 API
+            # Query ALL keys to find where Dynamo registers (empty key "" with range_end "\0" = all keys)
```

Shellcheck (0.11.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)